src/Components/Modals/Cards/index.js

```diff
@@ -45,67 +44,65 @@ class Cards extends PureComponent {
 
   render() {
     return (
-      <div className="card-settings-wrapper">
-        <div className="card-settings">
-          <h2>Card content</h2>
-          <ul>
-            <li>
-              <p>
-                <input
-                  type="checkbox"
-                  id="switch1"
-                  onChange={(e) => this.handleCheckboxChange(e, 'summary')}
-                  checked={this.state.cardContent.summary}
-                />
-                <label className="checkbox-label" htmlFor="switch1"/>
-                Show incident summary
-              </p>
-            </li>
-            <li>
-              <p>
-                <input
-                  type="checkbox"
-                  id="switch2"
-                  onChange={(e) => this.handleCheckboxChange(e, 'createdAt')}
-                  checked={this.state.cardContent.createdAt}
-                />
-                <label className="checkbox-label" htmlFor="switch2"/>
-                Show incident created at
-              </p>
-            </li>
-            <li>
-              <p>
-                <input
-                  type="checkbox"
-                  id="switch3"
-                  onChange={(e) => this.handleCheckboxChange(e, 'latestChange')}
-                  checked={this.state.cardContent.latestChange}
-                />
-                <label className="checkbox-label" htmlFor="switch3"/>
-                Show incident latest change
-              </p>
-            </li>
-            <li>
-              <p>
-                <input
-                  type="checkbox"
-                  id="switch4"
-                  onChange={(e) => this.handleCheckboxChange(e, 'changedBy')}
-                  checked={this.state.cardContent.changedBy}
-                />
-                <label className="checkbox-label" htmlFor="switch4"/>
-                Show incident latest change by
-              </p>
-            </li>
-          </ul>
-          <input
-            onClick={() => this.handleSave()}
-            className="submit"
-            type="submit"
-            value="Save"
-          />
-        </div>
-      </div>
+      <React.Fragment>
+        <h2>Card content</h2>
+        <ul>
+          <li>
+            <p>
+              <input
+                type="checkbox"
+                id="switch1"
```

Comment:
Using more descriptive names is highly recommended :slightly_smiling_face:  These are related to things that are named already, so I would use that, e.g. `summary-checkbox`.